packages/accordions/examples/advanced.md

```diff
@@ -0,0 +1,191 @@
+    sea lettuce brussels sprout cabbage. Catsear cauliflower garbanzo yarrow salsify chicory garlic
+    bell pepper napa cabbage lettuce tomato kale arugula melon sierra leone bologi rutabaga
+    tigernut. Sea lettuce gumbo grape kale kombu cauliflower salsify kohlrabi okra sea lettuce
+    broccoli celery lotus root carrot winter purslane turnip greens garlic. Jícama garlic courgette
+    coriander radicchio plantain scallion cauliflower fava bean desert raisin spring onion chicory
+    bunya nuts. Sea lettuce water spinach gram fava bean leek dandelion silver beet eggplant bush.`
+  }
+];
+
+const AdvancedExample = () => {
+  const [expandedSections, setExpandedSections] = React.useState([0]);
+  const [isBare, setIsBare] = React.useState(false);
+  const [isCompact, setIsCompact] = React.useState(false);
+  const [isExpandAll, setIsExpandAll] = React.useState(false);
+  const [isCollapseAll, setIsCollapseAll] = React.useState(false);
+  const onExpandAll = () => {
+    setIsExpandAll(!isExpandAll);
+    setIsCollapseAll(false);
+    setExpandedSections([0, 1, 2, 3, 4]);
+  };
+  const onCollapseAll = () => {
+    setIsCollapseAll(!isCollapseAll);
+    setIsExpandAll(false);
+    setExpandedSections([]);
+  };
+  const onChange = index => {
+    setExpandedSections(sections => [index]);
+    setIsExpandAll(false);
+    setIsCollapseAll(false);
+  };
+
+  const buttonGroup = (
+    <StyledButtonGroup>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <GearIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <FolderIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+    </StyledButtonGroup>
+  );
+
+  return (
+    <Grid>
+      <Well isRecessed>
+        <Row>
+          <Col>
+            <Field>
+              <Toggle checked={isBare} onChange={event => setIsBare(!isBare)}>
+                <Label>Bare</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isExpandAll} onChange={onExpandAll}>
+                <Label>Expand All</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isCollapseAll} onChange={onCollapseAll}>
+                <Label>Collapse All</Label>
+              </Toggle>
+            </Field>
```

Comment:
These two should probably be buttons – feels weird as toggle controls.

Reply:
Buttons are much more suitable for this 👍.